EFCore.SqlServer.DateOnlyTimeOnly/Query/SqlServerDateOnlyMethodTranslator.cs

```diff
@@ -69,6 +69,19 @@ public class SqlServerDateOnlyMethodTranslator : IMethodCallTranslator
                 instance.TypeMapping);
         }
 
+        if (method.DeclaringType == typeof(DateOnly)
+            && method.Name == nameof(DateOnly.FromDateTime)
+            && arguments.Count == 1)
+        {
+            return _sqlExpressionFactory.Function(
```

Comment:
Is there a particular reason to use CONVERT with a fragment, rather than just a standard Convert node (which translates to CAST)? I know we're already doing this elsewhere, just wondering if it can be simplified...

Reply:
No reason  just found it in EF Core codebase. Let me try convert

Reply:
@roji latest commit to master here is using Convert - thanks, looks much better 😄

Reply:
Great!